pkg/hash/hash.go

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
IMO better to have this in `openebs/api/pkg/util/hash.go` repo ?

Reply:
other way is i am going to raise a PR in api repo and we can import this later as well.